pkg/asset/installconfig/vsphere/validation.go

```diff
@@ -109,7 +109,7 @@ func validateNetwork(finder Finder, p *vsphere.Platform, fldPath *field.Path) fi
 		return field.ErrorList{field.Invalid(fldPath, p.Datacenter, err.Error())}
 	}
 	networkPath := fmt.Sprintf("%s/network/%s", dataCenter.InventoryPath, p.Network)
-	_, err = finder.Network(ctx, networkPath)
+	_, err = finder.NetworkList(ctx, networkPath)
```

Comment:
Should we do the same method of finding the network that we are doing in `getNetworkMoID` here to validate that the network exists in the specified cluster? Otherwise, this validation may give a false positive.

Reply:
This has been addressed in f10b387455edeb5314646abfe5f8231b8fa34e74 and required some rework to get unit tests (w/ mock) to run properly.